server/views/pages/bookAVisit/overbooking.njk

```diff
@@ -14,36 +15,34 @@
       <p>
         <span data-test="bookings-count">{{ bookingsCount }}</span> of
         <span data-test="max-capacity">{{ maxCapacity }}</span> {{ "table" | pluralise(maxCapacity) }} are booked for
-        <span data-test="visit-start-time">{{ visitStartTime }}</span> to
-        <span data-test="visit-end-time">{{ visitEndTime }}</span> on 
-        <span data-test="visit-date">{{ visitStartTime | formatDate("EEEE d MMMM") }}</span>.
+        <span data-test="visit-start-time">{{ visitSession.startTimestamp | formatTime }}</span> to
+        <span data-test="visit-end-time">{{ visitSession.endTimestamp | formatTime }}</span> on 
+        <span data-test="visit-date">{{ visitSession.startTimestamp | formatDate("EEEE d MMMM") }}</span>.
       </p>
+      <form action="{{ urlPrefix }}/confirm-overbooking" method="POST" novalidate>
```

Comment:
```suggestion
      <form action="{{ urlPrefix }}/check-your-booking" method="POST" novalidate>
```
Could just post directly to where you end up - then don't need the `POST /confirm-overbooking` route defining at all in `./routes/bookAVisit.ts`